<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>121</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="buttonFindNext">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="whatsThis">
       <string>Find next entry of the word</string>
      </property>
      <property name="text">
       <string>Find Next</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonReplace">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="whatsThis">
       <string>Replase next entry</string>
      </property>
      <property name="text">
       <string>Replace</string>
      </property>
      <property name="autoRepeatDelay">
       <number>294</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonReplaceAll">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="whatsThis">
       <string>Replace all entrys in the file</string>
      </property>
      <property name="text">
       <string>Replace All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="whatsThis">
       <string>Clsoe this dialog</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>dsds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Replace with:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="findWhat">
        <property name="whatsThis">
         <string>Find this</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="replaceWith">
        <property name="whatsThis">
         <string>Replase with this</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="matchCase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Mathc case while finding entrys</string>
   </property>
   <property name="text">
    <string>Match case</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
